classes/helper.php

```diff
@@ -549,6 +565,62 @@ public static function search_db(\tool_advancedreplace\search $record, string $o
         }
     }
 
+    /**
+     * Return a closure that can be used to create the link from the record.
+     *
+     * @param string $table   The name of the table being searched.
+     * @param string $column  The name of the column being searched.
+     * @return \Closure    $urlstring = closure($record).
+     */
+    public static function find_link_function($table, $column) {
```

Comment:
I'm seeing the warning 'find_link_function': not all code paths return a value, most likely because there's no return if it makes it to the end?